safe_transaction_service/history/models.py

```diff
@@ -1250,6 +1250,14 @@ def safes_pending_to_be_processed(self) -> QuerySet[ChecksumAddress]:
             .distinct("internal_tx___from")
         )
 
+    def safes_pending_to_be_processed_without_distinct(
+        self,
+    ) -> QuerySet[ChecksumAddress]:
+        """
+        :return: List of Safe addresses with repeated address that have transactions pending to be processed
+        """
+        return self.not_processed().values_list("internal_tx___from", flat=True)
```

Comment:
This is not sorted, so results are not stable. Any query can Safes in a total different order